lib/csv/parser.rb

```diff
@@ -675,7 +675,9 @@ def prepare_quoted
     def prepare_unquoted
       return if @quote_character.nil?
 
-      no_unquoted_values = "\r\n".encode(@encoding)
+      # Only exclude characters that are actually part of the row separator
+      # instead of hardcoding "\r\n"
```

Comment:
It seems that we can remove this comment.
I feel that it's useful for commit message (the PR description in this repository) because it describes why we do this change but it may not be useful for readers of new code. (Nobody will not try using `"\r\n"` here.)